e2e_helm_install_test failure on CI

<!--

This issue tracker is a best-effort forum for users and customers to report bugs.

Be sure to not include any sensitive information. Sensitive information should __NOT__ be included in this issue.

-->

**Bug description**:
e2e_helm_install_test e2e test failed in the CI running on main post PR merges:
https://github.com/openservicemesh/osm/runs/1646641466?check_suite_focus=true



**Affected area (please mark with X where applicable)**:
- Install                    [ ]
- SMI Traffic Access Policy  [ ]
- SMI Traffic Specs Policy   [ ]
- SMI Traffic Split Policy   [ ]
- Permissive Traffic Policy  [ ]
- Ingress                    [ ]
- Egress                     [ ]
- Envoy Control Plane        [ ]
- CLI Tool                   [ ]
- Metrics                    [ ]
- Certificate Management     [ ]
- Sidecar Injection          [ ]
- Logging                    [ ]
- Debugging                  [ ]
- Tests                      [X]
- CI System                  [ ]

**Expected behavior**: e2e test should pass

**Steps to reproduce the bug (as precisely as possible)**: Run the failed test

**How was OSM installed?**: Helm

Test output:
```
Running locally 'helm install helm-install-osm ../../charts/osm --set OpenServiceMesh.image.registry=localhost:5000,OpenServiceMesh.image.tag=40528178960354060bf4166c7ae6e0c05a6b90a9,OpenServiceMesh.meshName=helm-install-osm --namespace helm-install-namespace --create-namespace --wait'
Running locally 'helm uninstall helm-install-osm --namespace helm-install-namespace'
[AfterEach] [Top Level]
  /home/runner/work/osm/osm/tests/framework/common.go:68
STEP: [Cleanup] waiting for osmTest:1609799315 test NS cleanup

• Failure [18.540 seconds]
[Tier 2][Bucket 1] Test osm control plane installation with Helm
/home/runner/work/osm/osm/tests/framework/common.go:122
  Using default values
  /home/runner/work/osm/osm/tests/e2e/e2e_helm_install_test.go:16
    installs osm control plane successfully [It]
    /home/runner/work/osm/osm/tests/e2e/e2e_helm_install_test.go:17

    exit status 1
    

    /home/runner/go/pkg/mod/github.com/onsi/ginkgo@v1.14.1/internal/leafnodes/runner.go:113
------------------------------
SSSSDeleting kind cluster: osm-e2e


Summarizing 1 Failure:

[Fail] [Tier 2][Bucket 1] Test osm control plane installation with Helm Using default values [It] installs osm control plane successfully 
/home/runner/go/pkg/mod/github.com/onsi/ginkgo@v1.14.1/internal/leafnodes/runner.go:113

Ran 7 of 16 Specs in 651.361 seconds
FAIL! -- 6 Passed | 1 Failed | 0 Pending | 9 Skipped
--- FAIL: TestSuite (651.36s)
```

--- a/charts/osm/templates/validatingwebhook.yaml
+++ b/charts/osm/templates/validatingwebhook.yaml
@@ -25,6 +25,5 @@ webhooks:
       operations:
         - CREATE
         - UPDATE
-        - DELETE
       resources:
         - configmaps

